utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -115,23 +127,30 @@ impl StorageCmd {
 	{
 		let mut template = TemplateData::new(&cfg, &self.params);
 
+		let block_id = BlockId::<Block>::Number(client.usage_info().chain.best_number);
+		template.set_block_number(block_id.to_string());
+
 		if !self.params.skip_read {
 			let record = self.bench_read(client.clone())?;
-			record.save_json(&cfg, "read")?;
+			if let Some(path) = &self.params.json_read_path {
+				record.save_json(&cfg, &path, "read")?;
+			}
 			let stats = record.calculate_stats()?;
 			info!("Time summary [ns]:\n{:?}\nValue size summary:\n{:?}", stats.0, stats.1);
 			template.set_stats(Some(stats), None)?;
 		}
 
 		if !self.params.skip_write {
 			let record = self.bench_write(client, db, storage)?;
-			record.save_json(&cfg, "write")?;
+			if let Some(path) = &self.params.json_write_path {
+				record.save_json(&cfg, &path, "write")?;
+			}
```

Comment:
if `None`, we need to write it too (save_json could have Option<PathBuf> as param).

Reply:
This turns the behaviour around to only save the json files when we want to, by specifying `--json-read-path` or `--json-write-path`.  
I think that is better since we do not always need them.

Reply:
:+1: